lib/db.js

```diff
@@ -24,6 +24,20 @@ var Tasks = (module.exports.Tasks = require('./tasks')(db));
 var TaskUserStats = (module.exports.TaskUserStats = require('./task-user-stats')(
   db
 ));
+var ItemComments = (module.exports.ItemComments = require('./item-comments')(
+  db
+));
+var Tags = (module.exports.Tags = require('./tags')(db));
```

Comment:
Do you think we should call these `ItemTags` since we have `ItemComments`? I think it may help illustrate the relationship between items, tags, and comments.